src/main/resources/bidder-config/cpmstar.yaml

```diff
@@ -0,0 +1,24 @@
+adapters:
+  cpmstar:
+    enabled: false
+    endpoint: https://server.cpmstar.com/openrtbbidrq.aspx
+    pbs-enforces-gdpr: true
+    modifying-vast-xml-allowed: true
+    deprecated-names:
+    aliases:
+    meta-info:
+      maintainer-email: prebid@cpmstar.com
+      app-media-types:
+        - banner
+        - video
+      site-media-types:
+        - banner
+        - video
+      supported-vendors:
+      vendor-id: 0
+    usersync:
+      url: //server.cpmstar.com/usersync.aspx?gdpr={{gdpr}}&consent={{gdpr_consent}}&us_privacy={{us_privacy}}&redirect=
```

Comment:
change to  https: 
https://github.com/rubicon-project/prebid-server-java/pull/598